Management/src/ActuatorApi/Properties/launchSettings.json

```
@@ -0,0 +1,28 @@
+{
+  "$schema": "http://json.schemastore.org/launchsettings.json",
+  "profiles": {
+    "http": {
+      "commandName": "Project",
+      "launchBrowser": true,
+      "launchUrl": "swagger",
+      "environmentVariables": {
+        "ASPNETCORE_ENVIRONMENT": "Development",
+        "OTEL_EXPORTER_ZIPKIN_ENDPOINT": "http://localhost:9411/api/v2/spans"
+      },
+      "dotnetRunMessages": true,
+      // Steeltoe: Wildcard used instead of "localhost" here in order to support connections from local containers.
```

```suggestion
      // Steeltoe: Wildcard used instead of "localhost" here to support connections from local containers.
```
sync up with the other file